src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -429,10 +427,16 @@ class LicenceService(
   }
 
   @Transactional
-  fun inActivateLicences(licenceIds: List<Long>) {
+  fun activateLicencesByIds(licenceIds: List<Long>) {
```

Comment:
It's not strictly necessary in this case but you shouldn't shy away from it. 

sure it can be overused but it's idiomatic to use it. 

You have all the context you need from the method signature which is available everywhere